htdocs/install/mysql/migration/10.0.0-11.0.0.sql

```diff
@@ -48,8 +49,11 @@ UPDATE llx_c_units SET label = 'SurfaceUnitm2' WHERE code IN ('M2');
 
 ALTER TABLE llx_adherent_type ADD UNIQUE INDEX uk_adherent_type_libelle (libelle, entity);
 
-ALTER TABLE llx_mailing_cibles MODIFY COLUMN lastname varchar(160);
-ALTER TABLE llx_mailing_cibles MODIFY COLUMN firstname varchar(160);
+-- VMYSQL4.3 ALTER TABLE llx_mailing_cibles MODIFY COLUMN lastname varchar(160);
+-- VPGSQL8.2 ALTER TABLE llx_mailing_cibles ALTER COLUMN lastname TYPE varchar(160);
```

Comment:
This should not be required because it is job of the postgresql handler (method convertSQLFromMysql into class pqsql.class.php) to convert the request into mysql language.
See line 236:  if (preg_match('/ALTER TABLE ([a-z0-9_]+)\s+MODIFY(?: COLUMN)? ([a-z0-9_]+) (.*)$/i', $line, $reg)) ...